sonic_data_client/mixed_db_client.go

```diff
@@ -1122,11 +1125,7 @@ func (c *MixedDbClient) ConvertToJsonPatch(prefix *gnmipb.Path, path *gnmipb.Pat
 	}
 
 	elems := fullPath.GetElem()
-	if t == nil {
-		*output = `{"op": "remove", "path": "/`
-	} else {
-		*output = `{"op": "add", "path": "/`
-	}
+	*output = `{"op": "` + operation + `", "path": "/`
```

Comment:
>op [](http://example.com/codeflow?start=14&length=2)

Manual serialization is dangerous. Will you just construct the json object and use library to serialize it.